lib/fathead/scala/output.txt

```diff
@@ -0,0 +1,100 @@
+Nothing	A										<section class="prog_container"><p>Nothing is - together with scala.Null - at the bottom of Scala's type hierarchy.</p><p> Nothing is a subtype of every other type (including scala.Null); there exist\n no instances of this type.  Although type Nothing is uninhabited, it is\n nevertheless useful in several ways.  For instance, the Scala library defines a value\n scala.collection.immutable.Nil of type List[Nothing]. Because lists are covariant in Scala,\n this makes scala.collection.immutable.Nil an instance of List[T], for any element of type T.</p><p> Another usage for Nothing is the return type for methods which never return normally.\n One example is method error in scala.sys, which always throws an exception.\n</p></section>	https://www.scala-lang.org/api/current/scala/Nothing.html
+Tuple20	A										<section class="prog_container"><p>A tuple of 20 elements; the canonical representation of a scala.Product20.\n</p></section>	https://www.scala-lang.org/api/current/scala/Tuple20.html
+deprecatedName	A										<section class="prog_container"><p>An annotation that designates the name of the parameter to which it is\napplied as deprecated. Using that name in a named argument generates\na deprecation warning.</p><p>For instance, evaluating the code below in the Scala interpreter</p><pre>def inc(x: Int, @deprecatedName('y) n: Int): Int = x + n\ninc(1, y = 2)</pre><p>will produce the following output:</p><pre>warning: there were 1 deprecation warnings; re-run with -deprecation for details\nres0: Int = 3</pre></section>	https://www.scala-lang.org/api/current/scala/deprecatedName.html
+throws	A										<section class="prog_container"><p>Annotation for specifying the exceptions thrown by a method.\nFor example:</p><pre>class Reader(fname: String) {\nprivate val in = new BufferedReader(new FileReader(fname))\n@throws[IOException]("if the file doesn't exist")\ndef read() = in.read()\n}</pre></section>	https://www.scala-lang.org/api/current/scala/throws.html
+language	A										<section class="prog_container"><p> The scala.language object controls the language features available to the programmer, as proposed in the\n SIP-18 document.</p><p> Each of these features has to be explicitly imported into the current scope to become available:</p><pre>import language.postfixOps // or language._\nList(1, 2, 3) reverse</pre><p> The language features are:</p><pre> dynamics            enables defining calls rewriting using the Dynamic trait postfixOps          enables postfix operators reflectiveCalls     enables using structural typesimplicitConversions enables defining implicit methods and members higherKinds         enables writing higher-kinded types existentials        enables writing existential types experimental        contains newer features that have not yet been tested in production\n</pre></section>	https://www.scala-lang.org/api/current/scala/language$.html
```

Comment:
@sam09 it looks like we're turning a `<ul>` into a `<pre><code>` here?

https://www.scala-lang.org/api/current/scala/language$.html

We can put the `<ul>` into our `<section>` and it will work. If you add the class `prog__ul` to the `<ul>` then very soon, it will also display bullets. Right now our site CSS disables bullets on `<ul>`, but I'm adjusting that 👍